code/modules/reagents/reagent_containers/food/drinks/drinkingglass.dm

```diff
@@ -488,73 +489,73 @@
 				if(ICECOFFEE)
 					icon_state = "icedcoffeeglass"
 					item_state = "icedcoffeeglass"
-					name = "iced Coffee"
-					desc = "A drink to perk you up and refresh you!"
+					name = "iced coffee"
+					desc = "For when you need a coffee without the warmth."
 				if(COFFEE)
 					icon_state = "glass_brown"
 					item_state = "glass_brown"
 					name = "glass of coffee"
-					desc = "Don't drop it, or you'll send scalding liquid and glass shards everywhere."
+					desc = "Careful, it's hot!"
 				if(BILK)
 					icon_state = "glass_brown"
 					item_state = "glass_brown"
 					name = "glass of bilk"
-					desc = "A brew of milk and beer. For those alcoholics who fear osteoporosis."
+					desc = "A brew of milk and beer. For alcoholics who fear osteoporosis."
 				if(FUEL)
 					icon_state = "dr_gibb_glass"
 					name = "glass of welder fuel"
 					desc = "Unless you are an industrial tool, this is probably not safe for consumption."
 				if(BROWNSTAR)
-					icon_state = BROWNSTAR
-					item_state = BROWNSTAR
+					icon_state = "brownstar"
+					item_state = "brownstar"
 					name = "\improper Brown Star"
-					desc = "Its not what it sounds like..."
+					desc = "It's not what it sounds like..."
 				if(ICETEA)
-					icon_state = ICETEA
-					item_state = ICETEA
+					icon_state = "icetea"
+					item_state = "icetea"
 					name = "iced tea"
-					desc = "No relation to a certain rap artist/ actor."
+					desc = "Like tea, but refreshes rather than relaxes."
 				if(ARNOLDPALMER)
-					icon_state = ARNOLDPALMER
-					name = "Arnold Palmer"
-					desc = "Known as half and half to some.  A mix of ice tea and lemonade"
+					icon_state = "arnoldpalmer"
+					name = "\improper Arnold Palmer"
+					desc = "Known as half and half to some. A mix of ice tea and lemonade."
 				if(MILKSHAKE)
-					icon_state = MILKSHAKE
-					item_state = MILKSHAKE
-					name = MILKSHAKE
-					desc = "Glorious brainfreezing mixture."
+					icon_state = "milkshake"
+					item_state = "milkshake"
+					name = "milkshake"
+					desc = "Brings all the boys to the yard."
 				if(LEMONADE)
-					icon_state = LEMONADE
-					item_state = LEMONADE
-					name = LEMONADE
-					desc = "Oh the nostalgia..."
+					icon_state = "lemonade"
+					item_state = "lemonade"
+					name = "lemonade"
+					desc = "Oh, the nostalgia..."
 				if(KIRASPECIAL)
-					icon_state = KIRASPECIAL
+					icon_state = "kiraspecial"
 					name = "\improper Kira Special"
 					desc = "Long live the guy who everyone had mistaken for a girl. Baka!"
 				if(REWRITER)
-					icon_state = REWRITER
+					icon_state = "rewriter"
 					name = "\improper Rewriter"
-					desc = "The secert of the sanctuary of the Libarian..."
+					desc = "The librarian's special."
```

Comment:
The typo in that one was intentional, I believe.

Reply:
it's ancient baydescription, I doubt they'd do something that smart. The rewritten description is admittedly a bit bland too, so suggestions here are welcome.

Reply:
"The drink for crew members who can't read good and wanna learn to do other stuff good too."